test/unit/create-contentful-api-test.js

```diff
@@ -650,3 +688,52 @@ test('Given json should be parsed correctly as a collection of entries with meta
   t.looseEquals(parsedData.items[0].fields.metadata.sys, data.includes.Metadata[0].sys, 'metadata field is included')
   t.end()
 })
+
+test('resource links should be parsed correctly', (t) => {
+  const api = createContentfulApi({
+    http: {},
+    getGlobalOptions: sinon.stub().returns({ resolveLinks: true })
+  })
+  const data = {
+    items: [
+      {
+        sys: {
+          type: 'Entry',
+          locale: 'en-US'
```

Comment:
```suggestion

```

i'd remove just because `locale` doesn't exist in the object (`type` and `id` in fields should be enough since we are mocking it)